demo/app/android/app/src/main/kotlin/walletsdk/openid4ci/OpenID4CI.kt

```diff
@@ -28,8 +28,8 @@ class OpenID4CI constructor(
         newInteraction = Interaction(args, opts)
     }
 
-    fun authorize(): AuthorizeResult {
-        return newInteraction.authorize()
+    fun pinRequired(): Boolean {
```

Comment:
Note for @talwinder50: you'll want to rework how we check for the PIN as part of your upcoming PR to add support to the demo app for the authorization code flow - see `usage.md`